recipes/tcl/8.6.10/conanfile.py

```diff
@@ -85,8 +85,15 @@ def _patch_sources(self):
         win_makefile_vc = os.path.join(win_config_dir, "makefile.vc")
         tools.replace_in_file(win_makefile_vc, "@type << >$@", "type <<temp.tmp >$@")
 
+        win_rules_vc = os.path.join(self._source_subfolder, "win", "rules.vc")
         # do not treat nmake build warnings as errors
-        tools.replace_in_file(os.path.join(self._source_subfolder, "win", "rules.vc"), "cwarn = $(cwarn) -WX", "")
+        tools.replace_in_file(win_rules_vc, "cwarn = $(cwarn) -WX", "")
+        # disable whole program optimization
+        # to be portable across different MSVC versions.
```

Comment:
Let's leave a trail for our future selves =)

```suggestion
        # disable whole program optimization to be portable across different MSVC versions.
        # See conan-io/conan-center-index#4811 conan-io/conan-center-index#4094
```